doofinder-for-woocommerce/includes/api/class-store-api.php

```diff
@@ -51,6 +61,7 @@ public function __construct()
 
         $this->api_key = Settings::get_api_key();
         $this->api_host = Settings::get_api_host();
+        $this->store_helper = new Store_Helpers;
```

Comment:
Since the class is called "Store_Helpers" I think it would be better to name also the property as "$this->store_helpers". What do you think?

Reply:
Despite of this suggestion, I think the refactor has been great. Excellent work! :clap:

Reply:
You are right, a typo on my part! Thanks!